.github/workflows/lint-sample.yml

```diff
@@ -11,6 +11,13 @@ on:
 jobs:
   lint-samples:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        folder: [
```

Comment:
there are other test projects in docs folder, right?

Should they be added here as well?

Reply:
We are not linting the docs samples for the tutorial
We probably should to do that as well
We need an issue for it. See: https://github.com/kubernetes-sigs/kubebuilder/issues/4445